packages/commonwealth/client/scripts/views/components/archive_tray_with_tooltip.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { CWTooltip } from './component_kit/cw_popover/cw_tooltip';
```

Comment:
we should not use old tooltip anymore. We should use one from `new_designs`

Reply:
Done!